includes/wikia/models/DesignSystemGlobalNavigationModelV2.class.php

```diff
@@ -97,9 +55,34 @@ public function getData() {
 		return $data;
 	}
 
+	private function getMainNavigation() {
+		if ( $this->isWikiaOrgCommunity() ) {
+			return $this->getLink( self::COMMUNITY_CENTRAL_LABEL, $this->getHref('community-central'), self::COMMUNITY_CENTRAL_TRACKING_LABEL );
+		} else {
+			return array_merge(
+				$this->getFandomLinks(),
+				[
```

Comment:
how about extracting this array to method?

Reply:
makes sense